theano/tensor/tests/test_opt.py

```diff
@@ -6123,6 +6123,27 @@ def test3(self):
         assert len(divs) == 0
 
 
+def test_local_zero_div():
+    """Tests 0/x -> 0"""
+    mode = theano.compile.mode.get_default_mode().including("local_zero_div")
+    for t in (T.scalar, T.ivector, T.ftensor4):
+        x = t('x')
+        for op in (T.int_div, T.true_div, T.floor_div):
```

Comment:
floor_div is int_div, so only use one everywhere
scalar/basic.py have 

floor_div = int_div